middle_end/flambda2/to_cmm/to_cmm.ml

```diff
@@ -142,8 +142,12 @@ let unary_int_arith_primitive _env dbg kind op arg =
   | Naked_immediate, Swap_byte_endianness -> C.bswap16 arg dbg
   (* Special case for manipulating int64 on 32-bit hosts *)
   | Naked_int64, Neg when C.arch32 -> C.unsupported_32_bits ()
-  (* General case (including byte swap for 64-bit on 32-bit archi) *)
+  (* General case *)
```

Comment:
(I think this comment was meant for both
`| _, Neg` and `| _, Swap_byte_endianness`.)